x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/configuration_step_form.tsx

```diff
@@ -29,12 +28,12 @@ import {
 } from '@kbn/ml-data-frame-analytics-utils';
 import { DataGrid } from '@kbn/ml-data-grid';
 import { SEARCH_QUERY_LANGUAGE } from '@kbn/ml-query-utils';
+import type { OptionWithFieldStats } from '../../../../../components/field_stats_flyout/eui_combo_box_with_field_stats';
```

Comment:
Looks like there is still more work to do inside the DFA wizard as the width of the combo box isn't aligned with the width of the current input? Do we still need a full width control here?

<img width="661" alt="Screenshot 2024-06-25 at 10 10 52" src="https://github.com/elastic/kibana/assets/7405507/87cbe671-1f15-4c19-9047-91f0722ef53c">

<img width="1249" alt="Screenshot 2024-06-25 at 10 12 53" src="https://github.com/elastic/kibana/assets/7405507/e95a9018-9d8f-4287-9fae-451ed533799e">

Reply:
Updated in latest changes https://github.com/elastic/kibana/pull/186670/commits/57377aa718bee6ae9433735a086bb2395387620a